reference/UiTPAS.v2.json

```diff
@@ -2517,25 +2517,21 @@
             "authorizationUrl": "",
             "tokenUrl": "",
             "refreshUrl": "",
-            "scopes": {
-              "https://api.publiq.be/auth/uitpas": "Grants access to the UiTPAS API"
-            }
+            "scopes": {}
           }
         },
-        "description": "See [Authentication docs](https://publiq.stoplight.io/docs/authentication/docs/user-access-token.md)"
+        "description": "A user access token, obtained by redirecting the end user to publiq's authorization server to login. See the [authentication docs about user access tokens](https://publiq.stoplight.io/docs/authentication/docs/user-access-token.md) for more info."
       },
       "CLIENT_ACCESS_TOKEN": {
         "type": "oauth2",
         "flows": {
           "clientCredentials": {
             "tokenUrl": "",
             "refreshUrl": "",
-            "scopes": {
-              "https://api.publiq.be/auth/uitpas": "Grants access to the UiTPAS API"
-            }
+            "scopes": {}
           }
         },
-        "description": "See [Authentication docs](https://publiq.stoplight.io/docs/authentication/docs/client-access-token.md)"
+        "description": "A client access token, obtained by exchanging your client id and client secret for a token via an HTTP request to publiq's authorization server. See the [authentication docs about client access tokens](https://publiq.stoplight.io/docs/authentication/docs/client-access-token.md) for more info."
```

Comment:
HTTPS?

Reply:
Hmm personally I always just use "HTTP request" even though I mean HTTPS, because aside from the TLS part the rest of the standard is exactly the same if I'm not mistaken. The HTTP version in HTTP messages is even always denoted using just "HTTP" (for example "HTTP/3") even when doing an HTTPS request.

![Screen Shot 2022-01-14 at 13 39 59](https://user-images.githubusercontent.com/959026/149517279-eacbc1c4-30a4-4c24-a1e1-d0dadf79e4d9.png)

And maybe anecdotal, but I feel like most people just talk about "HTTP messages" in articles, talks etc about APIs.

I'm not against changing it to HTTPS here necessarily, but I think there are a lot of places in the docs where we already mention HTTP instead of HTTPS.

Maybe we could make it very explicit by adding a page about HTTPS in the Authentication docs, and mention that all our APIs only work over HTTPS, even if we somewhere mention HTTP as a general term?

Reply:
Here's also some indication that HTTPS not strictly a separate protocol.

> HTTP operates at the highest layer of the TCP/IP model—the application layer; as does the TLS security protocol (operating as a lower sublayer of the same layer), which encrypts an HTTP message prior to transmission and decrypts a message upon arrival. Strictly speaking, HTTPS is not a separate protocol, but refers to the use of ordinary HTTP over an encrypted SSL/TLS connection. 

https://en.wikipedia.org/wiki/HTTPS

But yeah it's true it only works over HTTPS specifically, but to me that's just part of the "HTTP" as a general term. (Not talking about `http://` vs `https://` here but more about the standard of messages)